### PR TITLE
CommonServerPython + ApiModules: Fixed insecure error message

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33205

## Description
don't suggest checking insecure param when already checked.

### CODE REVIEW

File: Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def http_request(self, method, url_suffix, full_url=None, headers=None, json_dat
                                     f'\n{res}')
 
         except Exception as e:
-            if 'SSL Certificate Verification Failed' in e.args[0]:
+            if self._verify and 'SSL Certificate Verification Failed' in e.args[0]:
```
Comment (Itay4) [inline Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.py:77]: shouldnt this be `if not self._verify` ?

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py [hunk 2]
```diff
@@ -2452,6 +2452,12 @@ def test_http_request_ssl_error(self, requests_mock):
         with raises(DemistoException, match="SSL Certificate Verification Failed"):
             self.client._http_request('get', 'event', resp_type='response')
 
+    def test_http_request_ssl_error_insecure(cls, requests_mock):
+        requests_mock.get('http://example.com/api/v2/event', exc=requests.exceptions.SSLError('test ssl'))
+        client = cls.BaseClient('http://example.com/api/v2/', ok_codes=(200, 201), verify=False)
+        with raises(requests.exceptions.SSLError, match="test ssl"):
+            client._http_request('get', 'event', resp_type='response')
```
Comment (Itay4) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:2459]: can we check here that the string "try selectin 'Trust any certificate'...' is not in the exception message?

File: Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.py (hunk 1 above)
Comment (yaakovi) [reply Packs/ApiModules/Scripts/ServiceNowApiModule/ServiceNowApiModule.py:77]: no, if the verify is true, it means that the checkbox is disabled (insecure is "not verify").

**Review by Itay4 — APPROVED** [review]

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py (hunk 2 above)
Comment (yaakovi) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py:2459]: updated the match string, now it will test for an exact match.

**Pull request merged**